ofl/leaguespartan/METADATA.pb

```diff
@@ -0,0 +1,23 @@
+name: "League Spartan"
+designer: "Tyler Finck"
```

Comment:
This is probably not complete. Since original author of Spartan is Matt Bailey, I think he should be including. I'll also ask Tyler if "The League Of" should be added as designer, as for Micah Rich.

Reply:
They didn't answer but I gonna merge anyway and they can claim changes later if they want.